docker/cpu_only/entrypoint.d/12-banner.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Do you know why these are being removed?

Reply:
I think I tried to over-standardize based on Cliff's comment [here](https://github.com/triton-inference-server/server/pull/6022#discussion_r1256428428). I wanted either all of those shell script files to be executable with shebangs or not executable without shebangs.

Thanks for bringing attention to this. I reverted all the changes so that they remain the same, just updating the executable bit to match the file's presence or absence of a shebang.